sysl2/sysl/ints_view_test.go

```diff
@@ -417,7 +417,7 @@ state "test" as X_1 {
   state "test" as _2
 }
 _0 -[#grey]-> _1
-_1 -[#black]> _2 : 
+_1 -[#black]> _2
```

Comment:
I dont like the fact that there was change in behaviour and only one test was broken.

Can you please add tests related to this specific functionality?

Reply:
@sahejsingh In fact, I was just deal with that format in wrong way, the ` : ` should be build within `label`.
But, I'll add a test case when `label` is not empty anyway.